sdks/java/harness/src/main/java/org/apache/beam/fn/harness/SplittableProcessElementsRunner.java

```diff
@@ -209,8 +199,22 @@ public void outputWindowedValue(
                 10000,
                 Duration.standardSeconds(10));
 
+    BoundedWindow window = elem.getWindows().iterator().next();
+    this.stateAccessor =
```

Comment:
You pass in the `stateAccessor` above which will be null.

You can follow the same pattern as in `FnApiDoFnRunner` where you create a stateAccessor once and track the current element/window.

Reply:
Fixed, thanks.